mmif/utils/__init__.py

```diff
@@ -1 +1,2 @@
 from mmif.utils import video_document_helper
+from mmif.utils import mmif_rewinder
```

Comment:
I think we can take of `mmif_` from module name and keep a concise name, since it's already under `mmif` top-level package.